src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+/* Styles */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+/* main header decoration: */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  width: 100%;
+  background-color: white;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+}
```

Comment:
Strange solution, you could just set vertical margins/paddings to parent container and use align-items